assets/js/vue/card.js

```diff
@@ -158,7 +172,7 @@ export default {
           </div>
 
           <div class="col-lg-4">
-            <div class="card-body d-flex flex-column h-100 p-4 p-lg-5">
+            <div class="card__body card-body d-flex flex-column h-100 p-4 p-lg-5">
```

Comment:
card__body als Klasse existiert nicht in der card.scss. Müsste card-body sein

Reply:
card-body kommt aus dem theme und card__body ist der Klassenname nach BEM. Mittelfristig wollen wir keine Styles aus dem Theme mehr nutzen und Elemente sollten auch konsequent BEM Klassennamen haben. card-body wird also irgendwann rausfliegen. Styling muss nicht existieren für einen Klassennamen. Ist sozusagen Vorbereitung für ein Refactoring, weil wir in dem Template eigentlich gerade 4 Templates haben.